enclave/core/hexdump.c

```diff
@@ -34,6 +34,12 @@ char* oe_hex_string(
     return str;
 }
 
+void oe_hex_dump_trace(oe_log_level_t level, const void* data, size_t size)
+{
+    OE_UNUSED(level);
+    oe_hex_dump(data, size);
```

Comment:
why dont you send level to this function and use it there?

Reply:
I have to extend this to enclaves too, similar to host/hexdump.c and host/traceh.c (oe_log_dump, etc). I am waiting to see if this direction is okay :-) Grep did not see a perf drop in enclave side of oe_verify_evidence since he did not include logging.edl in his .edl.